### DISCUSSION TITLE
Push Notifications By Pass Do Not Disturb on Android

### DISCUSSION THREAD

**Discussion by zburlott4** [airsdk/Adobe-Runtime-Support#3275] [General]
I understand Apple has a Critical Alerts entitlement that can be requested to by pass the do not disturb settings.  I also see instructions for this in the Air Native Extension for push notifications. 

I'm wondering if android has a similar mechanism to send alerts to by pass the dnd using the air native extension for android notifications.

**Comment by marchbold** [comment]
As far as I'm aware there's no direct equivalent of iOS's critical alerts on Android. The best you can do is set the "importance" of the notification channel to "high" and this will treat any notifications sent on that channel with the highest importance / priority. 

There are some workarounds but none that I have seen that work consistently. 

Additionally note that a user can configure a notification channel to override the Dnd state but currently a developer cannot override this setting ([reference](https://developer.android.com/reference/android/app/NotificationChannel#setBypassDnd(boolean)) )
